packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -828,22 +828,42 @@ class RenderListWheelViewport
 
   /// Paints all children visible in the current viewport.
   void _paintVisibleChildren(PaintingContext context, Offset offset) {
+    // Note that the magnifier cannot be turned off if the opacity is less than
```

Comment:
nit: per style guide just remove the "note that" at the beginning here.

Reply:
Done